src/routes/Weather/containers/WeatherContainer.js

```diff
@@ -0,0 +1,14 @@
+import { connect } from 'react-redux'
+import { forecast } from '../modules/weather'
+
+import Weather from 'components/Weather'
+
+const mapActionCreators = {
+  forecast: () => forecast('Belfast, uk')
+}
+
+const mapStateToProps = (state) => ({
+  weather: state.weather
+})
+
```

Comment:
can we create a Weather class here and in the `render` method put `<Weather {...this.props} />`?

It will introduce and enforce a better separation of concerns:
- view (Weather) doesn't need to know about redux and connect
- all the lifecycle methods will be in the container, hence we need class here